pkg/provider/resource_region.go

```diff
@@ -25,7 +25,7 @@ func NewRegionResource() *RegionResource {
 
 func (r *RegionResource) Schema() *schema.Resource {
 	return &schema.Resource{
-		Description: "Manage a region",
+		Description: "The region resource is used to manage regions for a given cloud provider. See https://www.enterprisedb.com/docs/biganimal/latest/getting_started/activating_regions/ for more details.",
```

Comment:
```suggestion
		Description: "The region resource is used to manage regions for a given cloud provider. See [Activating regions](https://www.enterprisedb.com/docs/biganimal/latest/getting_started/activating_regions/).",
```

Reply:
Clickable link is added at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/0aea7392df32d1d3b7fcb6e35524aa585fa8a660